cinema_service/settings.py

```diff
@@ -86,8 +91,12 @@
 
 DATABASES = {
```

Comment:
You are using environment variables to configure your database settings, which is a good practice. However, you have provided default values for these settings. It's recommended to not provide default values for sensitive data like database name, user, password, host and port. If these environment variables are not set, it's better to let the application fail to start, rather than it starting with potentially insecure default settings.